l10n_br_nfe/models/res_partner.py

```diff
@@ -89,16 +96,19 @@ def _compute_nfe40_enderDest(self):
             rec.nfe40_enderDest = rec.id
 
     @api.multi
+    @api.depends('company_type', 'cnpj_cpf')
```

Comment:
Aqui no depends deveria colocar o campo country_id, e se for um cliente de outro pais preencher o nfe40_choice7 com nfe40_idEstrangeiro

Reply:
@renatonlima Corrigido!